### PR TITLE
d3-geo > remove CoordinateReferenceSystem (crs)

### PR DESCRIPTION
**Pull request by j4ys0n**
https://tools.ietf.org/html/rfc7946#appendix-B

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://tools.ietf.org/html/rfc7946#appendix-B
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/d3-geo/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
-// Type definitions for D3JS d3-geo module 1.9
+// Type definitions for D3JS d3-geo module 1.10
```
Comment (gustavderdrache) [inline types/d3-geo/index.d.ts]: Whoops, careful! The d3-geo module is still only at [1.9](https://github.com/d3/d3-geo/releases). This version number tracks the upstream module, not the type definitions. Updates are pushed out as patch bumps.

**Comment by gustavderdrache** [comment]
It looks like this might duplicate #21801.

File: types/d3-geo/index.d.ts (hunk 1 above)
Comment (j4ys0n) [reply types/d3-geo/index.d.ts]: gotcha. i tried 1.9.1 but travis wouldn't allow patch versioning. thought it was appropriate to update the version number to indicate a change.

Comment (j4ys0n) [reply types/d3-geo/index.d.ts]: reverted. doesn't matter if we merge this one or the existing PR below. but this is breaking our build. whatever is faster would be preferred. :)

**Comment by j4ys0n** [comment]
Any chance we can get this PR approved so I can merge it in? I think the author of the duplicate PR is done for the day (he's in Germany). We need to get our build running again and this is the holdup. 
Thanks!

**Comment by tomwanzek** [comment]
👎 Favoring merging #21801, it's already approved. Pure logistics. No need for the original author of that PR to make changes there right now. So we can expedite the change by simply getting a maintainer to perform the merge.

Regardless of that, thanks to @j4ys0n !

**Comment by tomwanzek** [comment]
@j4ys0n Please close this PR to avoid confusion going forward.

We have just **released an interim patch** for the **d3-geo** TS definitions (version 1.9.4). Until completion of an impact assessment and resulting re-factoring of **d3-geo**, the patch replaced the dependency  on `CoordinateReferenceSystem` with a literal. I.e., _for now_, it  retains the `crs` optional property of the `ExtendedGeometryCollection<GeometryType extends GeoGeometryObjects>` interface as a conservative strategy.

I am reviewing whether we should start a new issue to consider the change impacts and refactoring, or re-purpose one  of the already open ones.

Thanks!!!

**Comment by mhegazy** [comment]
Thanks for your contribution. This PR has merge conflicts and can not be merged in at the time being. For housekeeping purposes we are closing stale PRs. If you'd still like to merge this code in, please open a new PR that has been merged and rebased with the `master` branch.

**Pull request closed without merging**